### PR TITLE
:pencil2: Fixes violations, translations + minor UI tweaks

### PR DESCRIPTION
**Pull request by QuintonC**
⚠️ Part of delegate.cash work.

## What is the context for these changes?

Some additional polish 💅 on the delegate cash work.

Primarily, this addresses:
- ESLint violations
- Translation concerns
  - Translations should not be split into numerous strings because sentence structures rely on context before and after a breakpoint, thus this could lead to incorrect translations.
- Minor UI tweaks (specifically with the underlines and usage of `Text` in styled components)
- SVG fixes (`fill-rule` ➡️ `fillRule`)

## Demonstration

...

## How can this be tophatted?

...

## 🎩 Checklist

- [ ] Tested on mobile
- [ ] Tested on multiple browsers
- [ ] Tested for accessibility
- [ ] Updated relevant documentation for the changes (if necessary)

### CODE REVIEW

**Review by imjameshall — APPROVED** [review]
This all looks good to me, would wait on @Soleone to put some eyes on it though, as it touches more parts that he's familiar with.

thank you!

File: packages/connect-wallet/src/components/ConnectButton/Popover.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {ConnectorIcon} from '../ConnectorIcon';
 import {useAppSelector} from '../../hooks/useAppState';
 import {useDisconnect} from '../../hooks/useDisconnect';
 import {useTranslation} from '../../hooks/useTranslation';
-import {DelegatedWalletsButton} from '../DelegatedWalletsButton';
+import {DelegatedWalletsDetail} from '../DelegatedWalletsDetail';
```
[the remark this answers is not in the document]
Comment (Soleone) [reply packages/connect-wallet/src/components/ConnectButton/Popover.tsx:16]: Right, thanks. It used to be a button that expanded the area further below it. But was simplified.

File: packages/connect-wallet/src/components/Modal/Screens/WhatAreDelegatesScreen.tsx [hunk 2]
```diff
@@ -26,10 +29,15 @@ const WhatAreDelegatesScreen = () => {
...
           <ListItemContent>
             <Text as="p" color="secondary">
-              {t('WhatAreDelegates.home.linkYourWallet')}
-              <a href='https://delegate.cash' target="_blank">delegate.cash</a>
-              &nbsp;&ndash;&nbsp;
-              {t('WhatAreDelegates.home.includesTransactionFee')}
+              <Trans i18nKey="WhatAreDelegates.home.content" t={t}>
+                <a
+                  href="https://delegate.cash"
+                  rel="noreferrer"
```
Comment (Soleone) [inline packages/connect-wallet/src/components/Modal/Screens/WhatAreDelegatesScreen.tsx:35]: Curious, why omit the referrer? Isn't it nice info for delegate.cash where users are coming from?

**Review by Soleone — APPROVED** [review]
Thanks a lot, really appreciate the attention to detail!

File: packages/connect-wallet/src/components/Modal/Screens/WhatAreDelegatesScreen.tsx (hunk 2 above)
Comment (QuintonC) [reply packages/connect-wallet/src/components/Modal/Screens/WhatAreDelegatesScreen.tsx:35]: It's an ES Lint rule
image

**Pull request merged**